README.md

```diff
@@ -43,6 +43,51 @@ The project uses woodpecker to automate builds. The configuration files can be f
 
 
 ## Reference
+
+### Feature flags
+
+Feature flags are used to enable/disable features in the application. They are defined in [config/environment.js](config/environment.js). 
+
+```javascript
+// in config/environment.js
+let ENV = {
+    // Other configuration settings...
+    features: {
+      'new-feature': true,    // Enable the 'new-feature' by default
+      'beta-feature': false,  // Disable the 'beta-feature' by default
+    }
+  };
+```
+
+The configuration can be manually overridden by setting a variable in the browser console, feature name have to be prefixed with `feature-`: 
```

Comment:
You right, a piece was missing. In the new version, you can override the configuration through query parameters. The overriding will be saved in a cookie, so it will persist across page reloads. The cookie can be cleared by adding `?clear-feature-overrides=true` to the URL.